src/components/Download/Download.jsx

```diff
@@ -0,0 +1,155 @@
+import React, { useState, useEffect } from "react";
+import { Table, NavLink, Input, Row, Col } from "reactstrap";
+import { IconContext } from "react-icons";
+import { FaDownload } from "react-icons/fa";
+import "./Download.css";
+
+export default () => {
+  const [Platform, setPlatform] = useState("");
+
+  useEffect(() => {
+    const availableOs = ["Win32", "Linux x86_64", "MacIntel"];
+    const detectedOs = window.navigator.platform;
+    if (availableOs.includes(detectedOs)) {
+      setPlatform(detectedOs);
+    } else {
+      setPlatform("Win32");
+    }
+  }, []);
+
+  const download = async () => {
```

Comment:
I just got to a point , it would be helpful to handle errors too ?

Like it is not sure if when the given url willl 100% give 200 0k status or if the client is having network issue , then there must be error handling system to handle those cases

@majordwarf @pollend @ryuk156 what are your thoughts ?

Reply:
@endurance21 you're right will soon make changes thanks.